### DISCUSSION TITLE
Downloads Link Directs To Prusa Slicer Website?

### DISCUSSION THREAD

**Discussion by HikariNoKitsune** [supermerill/SuperSlicer#3348] [Q&A] [answered]
Hey uh.... I was using 2.4.58.3, and was about to slice something... then it prompted me that there was an update... so, like usual I clicked the "Downloads" link given, and it... takes me to the Prusa Slicer Website??? Was this intentional? I thought "oh well maybe they somehow merged?" and went to download it, and then forgot that the entire reason I'm USING SuperSlicer is because it supports Klipper and my Printer........ So I manually search for SuperSlicer and find that it legit did have an update 24 days ago...

Am I just being weird here, or was that intentionally meant to take me to a 'competitor'?

**Comment by neophyl** [comment, marked as answer]
https://github.com/supermerill/SuperSlicer/issues/3262

**Reply by HikariNoKitsune** [reply]
Awesome Thank ya, did not see this when searching.
